account_invoice_negative_total/tests/test_account_invoice_open.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+from odoo.tools import float_compare
+
+
+class TestInvoiceNegativeAmount(TransactionCase):
+    def setUp(self):
```

Comment:
Let's use SavepointCase with setUpClass

Reply:
I tried using setupclass, it breaks the tests. I don't want to spend time fixing tests just to use setupclass.